e2e/transform/transform-snapshotResolver/snapshotResolver.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {SnapshotResolver} from 'jest-snapshot';
+
+const snapshotResolver: SnapshotResolver = {
+  // resolves from test to snapshot path
+  resolveSnapshotPath: (testPath, snapshotExtension) =>
+    testPath.replace('__tests__', '__snapshots__') + snapshotExtension,
```

Comment:
this is the default - maybe create something different to really show it's working? we don't need the comments either

Reply:
if by "default" you mean the default example of a custom snapshot resolver, than yes.

This will result in snapshots being outputted next to `__test__` rather than in said folder.

I figured that would be good enough since we're not trying to test that the actual snapshot resolving is working; only that the custom resolver is being transformed.

Reply:
Subtle ^^ Missed that as well at first.

Reply:
In my defence, there were comments... 😜

I'm happy to change it to something less subtle if you'd prefer, but figured it was nice to have the exact TS version of the example from the docs 🙂

Reply:
@SimenB Don't forget to let me know if you'd like a less subtle example, or resolve if you're happy 😃